src/android/CDVMFPAnalytics.java

```diff
@@ -58,30 +58,18 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         }
         return false;
     }
-
+    
     /**
      * Sends the Analytics logs to the server
      * @param callbackContext Callback that will indicate whether the request succeeded or failed
      */
     public void send(final CallbackContext callbackContext) {
         cordova.getThreadPool().execute(new Runnable() {
             public void run() {
-                Logger.sendAnalytics(new ResponseListener() {
-                    @Override
-                    public void onSuccess(Response response) {
-                        callbackContext.success();
-                    }
-
-                    @Override
-                    public void onFailure(Response failResponse, Throwable t, JSONObject extendedInfo) {
-                        callbackContext.error(failResponse.toString());
-                    }
-
-                });
+                Logger.send();
```

Comment:
This line is wrong; two things: first it should be `Analytics.send()`, second, you should pass the same response listener as before; otherwise the Cordova callback never gets called.